x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/EvaluatorImplementer.java

```diff
@@ -114,20 +131,30 @@ private MethodSpec ctor() {
         return builder.build();
     }
 
+    @SuppressWarnings("checkstyle:DescendantToken")
     private MethodSpec eval() {
         MethodSpec.Builder builder = MethodSpec.methodBuilder("eval").addAnnotation(Override.class);
         builder.addModifiers(Modifier.PUBLIC).returns(BLOCK).addParameter(PAGE, "page");
-
+        if (mv) {
+            // The first two parameters(builder, position) are place holders, no need to resolve them
+            processFunction.args.remove(0);
+            processFunction.args.remove(0);
```

Comment:
I'm not comfortable mutating the list here - but you could do it in the constructor.